### PR TITLE
Tweet about new glitch README

### PR DESCRIPTION
**Pull request by tcbyrd**
Deployed the updated glitch README that shows off how easily you can get started now, so figured we should 🐦 tweet about it.

cc @melissamcewen @gr2m for 👀 and 👍

### CODE REVIEW

File: tweets/glitch-update-15-03-19.tweet [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Thanks to @melissamcewen for providing a shiny new README that shows off how easily you can get started writing GitHub Apps with @Glitch!!
```
Comment (gr2m) [inline tweets/glitch-update-15-03-19.tweet]: Maybe "... writing Probot Apps with @Glitch!!" or "... writing GitHub Apps with @Glitch and Probot!!"?

I’d also just do one "!", someone told me once that multiple exclamation marks reads like SCREAMING :)

Comment (gr2m) [reply tweets/glitch-update-15-03-19.tweet]: oh "with me" is even nicer 👉🤖👈

**Comment by tcbyrd** [comment]
@gr2m Thoughts on waiting for a 👍from @melissamcewen before merging?

**Comment by melissamcewen** [comment]
Awesome! Thanks so much! We plan to publish some stuff on this on the front page of Glitch tomorrow. If there is any Probot related art that y'all would like to feature, we could use that or just the regular logo. Also here is our accompanying blog post

> # **Meet Your Github BFF: Probot the Robot**

Are your Github projects disorganized? Have a bunch of open issues you haven't gotten to yet, or forgot to close? 20 old branches like "feature-request-new-header-2016"? I know I have all these problems, which is why I was delighted to discover Probot.

Probot is a bot tailor-made for Github: It can handle your Github-related chores—and even help with project management and community-building tasks like polls, sharing, and welcoming new contributors.

Check out a list of some of the great Probots people have built on their [Featured Page](https://probot.github.io/apps/).

So how do you get your very own Github Probot? Glitch is the perfect place to make one, and now it's easier than ever. We teamed up with Probot to update our "Hello World" Probot app.

EMBED:

[https://glitch.com/~probot-hello-world](https://glitch.com/~probot-hello-world)

Before today, getting started with Probot was required some time and attention to detail. You had to go back and forth between Glitch and Github over and over again. Now most of that is done with one button click. No more copying keys or webhooks, and the "app manifest" (see app.yml) configures permissions for you.

That means more bots to do your bidding! What can they do? We spun up a couple of popular ones on Glitch to give you some ideas:

- Github Stale - automatically marks inactive issues as stale
- Probot Todo- turns todos into your code into new issues so you don't forget about that "@todo" buried in some random file

We also created a new one just for Glitch! Open an issue here in this repo and you'll get a special welcome message featuring rainbow text and cats.

[https://github.com/melissamcewen/probot-meow-welcome](https://github.com/melissamcewen/probot-meow-welcome)

OK maybe rainbow text and cats aren't for every project, but it also is a good starting place for a bot that can utilize user info from Github or make images programmatically.

We're also featuring a couple of community built picks in our Probot Collection. Check them out:

[https://glitch.com/@glitch/probot](https://glitch.com/@glitch/probot)

Make a cool Probot? Share it with us by tweeting us at @Glitch.

**Comment by gr2m** [comment]
Can we give feedback on the blog post? Maybe we can link to the post in the tweet once it’s published?

> Probot is a bot tailor-made for Github: It can handle your Github-related chores—and even help with project management and community-building tasks like polls, sharing, and welcoming new contributors.

I think this might makes it sound as if Probot was a product itself, while it is a framework to build GitHub Apps which in turn can take care of the mentioned chores.

> We spun up a couple of popular ones on Glitch to give you some ideas

You can also spun https://github.com/wip/app if you like, I made it in way that it can be used as a starting point. And maybe link to the apps in the list?

---

On https://glitch.com/@glitch/probot, capitalize the "H" in "Probot: Your BFF on Github!" and "Probot is here to help you on Github! 🤖" ;)

**Comment by tcbyrd** [comment]
I missed this before, but in general we want to make sure the `H` in GitHub is always capitalized. I missed it on the hello world, but updated it just yesterday.

> Github Stale

I wouldn't recommend this one since it only works if the underlying app runs continuously. On Glitch this won't work since once the container sleeps, the scheduled job that looks for stale issues will not run. Gregor's recommendation of WIP is a great fit for Glitch, though!

**Comment by melissamcewen** [comment]
Thanks for the feedback, I'll make those edits tonight and switch out stale
for wip

**Comment by melissamcewen** [comment]
Maybe I should open in WIP but I wasn't sure what permissions to use? I made an app.yml with the ones that seemed to work unless that's not recommended for this one https://glitch.com/edit/#!/probot-wip-glitch

**Comment by melissamcewen** [comment]
Also should probably update the Glitch Readme with GitHub instead of Github.

**Comment by gr2m** [comment]
WIP needs the following permissions

- checks: read & write
- Pull requests: read & write
- Single file: read only for `.github/wip.yml`

For events it needs

- check_run
- pull_request

We should add an app.yml to WIP :)

**Comment by melissamcewen** [comment]
Cool well here is a start, not sure where the single file part goes since it wasn't in that template https://github.com/wip/app/pull/182

**Comment by gr2m** [comment]
Merged 👍

**Comment by melissamcewen** [comment]
New blog post up: https://glitch.com/culture/meet-your-github-bff-probot-the-robot/

**Comment by tcbyrd** [comment]
Thanks! @gr2m I updated the wording a bit and added a link to the blog post. If you're 👍 with the changes let's merge it!

**Pull request merged**